app/home/controllers/home-controller.coffee

```diff
@@ -8,9 +8,24 @@
 
 ###
 class HomeCtrl
-  constructor: ->
+  $inject = ['AuthFactory1', '$mdSidenav']
+  constructor: (@AuthFactory1, @$mdSidenav) ->
     @ctrlName = 'HomeCtrl'
-
+    @logout = @AuthFactory1.logout
+    @settings = [
+      { name: 'Dashboard', enabled: true, icon: 'fa fa-book', route: 'home.dashboard' },
+      { name: 'Exams', enabled: true, icon: 'fa fa-file-text-o', route: 'home.exams' },
+      { name: 'Stats', enabled: false, icon: 'fa fa-bar-chart', route: 'home.stats' },
+    ]
+  close: () ->
+    @$mdSidenav('left').toggle()
+  toggle: () ->
+    @$mdSidenav('left').toggle()
+  isLoggedIn: (num) ->
```

Comment:
if you want to do this just use another method to say isNotLoggedIn or just negate isLoggedIn in the template